src/petals/cli/run_server.py

```diff
@@ -78,11 +77,9 @@ def main():
     parser.add_argument("--torch_dtype", type=str, default="auto",
                         help="Use this dtype to store block weights and do computations. "
                              "By default, respect the dtypes in the pre-trained state dict.")
-    parser.add_argument('--attn_cache_size', type=str, default=None,
-                        help='The size of GPU memory allocated for storing past attention keys/values between inference steps. '
-                             'Examples: 500MB, 1.2GB, 1073741824 (bytes). Note that 1KB != 1KiB here. '
-                             'Default: 0.5GiB * num_blocks * hidden_size / 14336. '
-                             'The latter is the hidden size of the bigscience/bloom-petals model.')
+    parser.add_argument('--attn_cache_size', type=int, default=8192,
```

Comment:
```suggestion
    parser.add_argument('--attn_cache_tokens', type=int, default=8192,
```

Let's rename it to **(1)** reflect that it's measured in tokens (in contrast to other options ending with `size`) and **(2)** avoid collisions (previously, int values were also correct and meant bytes - we need to show the error if someone used it).